src/ml_tooling/plots/__init__.py

```diff
@@ -5,6 +5,7 @@
 from .prediction_error import plot_prediction_error
 from .residuals import plot_residuals
 from .roc_auc import plot_roc_auc
+from .precesion_recall_curve import plot_pr_curve
```

Comment:
```suggestion
from .precision_recall_curve import plot_pr_curve
```

Lille stavefejl :smile: